build-tools/knnplugin-coverage.gradle

```diff
@@ -74,7 +74,7 @@ allprojects{
         jacocoTestReport.dependsOn integTest.runner
 
         testClusters.integTest {
-            jvmArgs " ${dummyIntegTest.jacoco.getAsJvmArg()}"
+            jvmArgs " ${dummyIntegTest.jacoco.getAsJvmArg()}".replace('javaagent:','javaagent:/')
```

Comment:
Just curious, what does this do?

Reply:
Fixes jacocoagent path to work with gradle 6.5 for integration tests